modules/manage/pages/recovery-mode.adoc

```diff
@@ -0,0 +1,201 @@
+
+Brokers can only enter recovery mode as it starts up, and not while it is already running. First set the broker configuration property to enable recovery mode, and then do a broker restart.
+
+== Use recovery mode
+
+To use recovery mode, first configure the broker to enable recovery mode when a broker starts.
+
+. Run the `rpk redpanda mode` command to set the `recovery_mode_enabled` broker configuration property to `true`.
+// TODO: confirm whether rpk reference is missing this update
++
+[,bash] 
+---- 
+rpk redpanda mode recovery
+----
+
+. Start the cluster. You can run `rpk cluster health` to check whether the cluster has entered recovery mode.
++
+It is possible to start a mixed-mode cluster, where some individual brokers are in recovery mode while others are not, but Redpanda recommends starting all brokers in the cluster in recovery mode together.
+
+All private Redpanda topics such as `__consumer_offsets` are accessible. While data in user-created topics is not available, you can manage the topics metadata.
+
+In recovery mode, you can also <<disable partitions,disable partitions>> if the cluster failure is localized to just a few partitions. If you disable specific partitions while in recovery mode, the disabled state persists even when you restart Redpanda again normally.
+
+You can perform the following operations in recovery mode:
+
+* Edit controller metadata
+** Edit cluster configuration properties
+** Add and remove users and ACLs
+** Add new brokers to the cluster
+** Modify topic properties
+** Delete topics
+** Delete WASM transforms
+* Edit consumer group metadata
+
+Node-wide and cluster-wide processes are disabled as they may disrupt recovery operations:
+
+* Partition and leader balancers
+* Tiered storage object storage housekeeping
+* Tiered storage cache management
+* Compaction
+
+You can use the Admin API when the cluster is in recovery mode. The HTTP Proxy and Schema Registry APIs are not available.
+
+== Disable partitions
+
+You may find that problems preventing normal cluster startup are often isolated to certain partitions or topics. In these situations, you can use the Admin API to manage these partitions. You can disable partitions at the topic level, or individual partition level. 
+
+You can also disable partitions outside of recovery mode.
+
+If you disable partitions while in recovery mode, starting Redpanda again in non-recovery mode leaves these partitions in a deactivated state. Disabled partitions and topics return a "Replica Not Available" error code for Kafka API requests.
+
+See the examples below for using the Admin API to enable or disable partitions. The examples assume that the Admin API port is `8081`.
+
+=== Disable a specific partition of a topic
+
+// TODO: Confirm whether these are new API endpoints and that they are available
+[tabs]
+====
+Curl::
++
+--
+```bash
+curl -X POST -d '{"disabled": true}' http://localhost:8081/cluster/partitions/<namespace>/<topic_name>/<partition_id>
```

Comment:
I guess we should mention that the namespace for user topics will be `kafka`. Or just insert it into the urls directly.